core/db_models/EEM_Payment_Method.model.php

```diff
@@ -268,25 +358,24 @@ public function verify_button_urls($payment_methods = null)
                 // though this is a caffeinated install, reset it to the default.
                 $current_button_url = $payment_method->button_url();
                 if (empty($current_button_url)
-                || (
+                    || (
                         strpos($current_button_url, 'decaf') !== false
                         && strpos($payment_method->type_obj()->default_button_url(), 'decaf') === false
                     )
                 ) {
                     $payment_method->save(
                         [
-                            'PMD_button_url' => $payment_method->type_obj()->default_button_url()
+                            'PMD_button_url' => $payment_method->type_obj()->default_button_url(),
                         ]
                     );
                 }
             } catch (EE_Error $e) {
-                $payment_method->set_active(false);
+                $payment_method->deactivate();
```

Comment:
the `set_active()` method is not a regular setter and you can't just pass it `flase` because it does NOT even have any parameters!!!
so if you want to deactivate a payment method you need to call the `deactivate()` method